### DISCUSSION TITLE
使用群晖chat如何填写自定义推送中的三个项目？

### DISCUSSION THREAD

**Discussion by EdisonCh4n** [qd-today/qd#422] [Q&A] [answered]
如题，纯小白，有没有大佬打个样？

**Comment by a76yyyy** [comment, marked as answer]
参考 [官方说明文档](https://kb.synology.com/zh-tw/DSM/tutorial/How_to_configure_webhooks_and_slash_commands_in_Chat_Integration)及议题 qd-today/qd#314, 群晖chat webhook实现自定义推送中三个项目可能的内容形式如下:

1. **URL**: https://**群晖地址**/webapi/entry.cgi?api=SYNO.Chat.External&**群晖chat_token**
2. **header**: 不填写任何内容
3. **data**: `payload={"text": "{log}"}`
4.

**Reply by EdisonCh4n** [reply]
谢谢，已成功，原来是默认登录次数错误过多导致ip被封发送失败！
